doc/whats_new.rst

```diff
@@ -24,6 +24,10 @@ Changelog
 
     - Add system config utility :func:`mne.sys_info` by `Eric Larson`_
 
+    - Automatic cross-validation and scoring metrics in :func:`mne.decoding.GeneralizationAcrossTime`, by `Jean-Remi King`_
+
+    - :func:`mne.decoding.GeneralizationAcrossTime` accepts non-deterministic cross-validations, by `Jean-Remi King`_
```

Comment:
I'm only aware of ShuffleSplit and ShuffleLabelSplit, but I'm not sure. the script now supports any non deterministic cv.